service/controller/release/resource/apps/resource.go

```diff
@@ -105,6 +122,12 @@ func (r *Resource) ensureState(ctx context.Context) error {
 
 	appsToCreate := calculateMissingApps(components, apps)
 	for _, app := range appsToCreate.Items {
+		if !key.AppConfigDeployed(app, configs) {
+			// Skip this app
+			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("skipping app %#q as its config is not deployed", app.Name))
+			continue
```

Comment:
I would be interested how much longer it takes like this for apps to be created. I would expect a delay of ~2-5 minutes at maximum which is acceptable to me.

Reply:
I've tested on kind cluster only because we don't have config-controller config controller ready (omg that naming). TIL status update is not CR update in sense of what operator reconciles

Reply:
Creating config takes ~10-15s with cold cache.

Reply:
> TIL status update is not CR update in sense of what operator reconciles

Yes that's a different resource. I don't know if you can watch that.